### PR TITLE
Release aristotle 3.0.4

### PR DESCRIPTION
**Pull request by harrywhite4**

### CODE REVIEW

File: python/aristotle-metadata-registry/aristotle_mdr/forms/bulk_actions.py [hunk 1]
```diff
@@ -211,9 +207,8 @@ def make_changes(self):
             tag__profile=self.user.profile,
             item__in=list(items)
         )
-        num_favourites = favourites.count()
         favourites.delete()
-        return _('%(num_items)s items removed from favourites') % {'num_items': num_favourites}
+        return _('{} items removed from favourites.'.format(favourites.count()))
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/forms/bulk_actions.py]: Revert plz.

File: python/aristotle-metadata-registry/aristotle_mdr/forms/bulk_actions.py [hunk 2]
```diff
@@ -230,23 +225,14 @@ def can_use(cls, user):
...
-    def generate_moving_message(org_name, sucessfully_moved_items: int, failed_items=None) -> str:
+    def generate_moving_message(org_name, successfully_moved_items: int, failed_items=None) -> str:
         if not failed_items:
-            message = _(
-                "%(num_items)s items moved into the workgroup '%(new_wg)s'. \n"
-            ) % {
-                'new_wg': org_name,
-                'num_items': sucessfully_moved_items,
-            }
+            message = _("{} items moved into the workgroup '{}'.".format(successfully_moved_items, org_name))
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/forms/bulk_actions.py]: Revert plz.

File: python/aristotle-metadata-registry/aristotle_mdr/forms/bulk_actions.py [hunk 3]
```diff
@@ -230,23 +225,14 @@ def can_use(cls, user):
...
         else:
             message = _(
-                "%(num_items)s items moved into the workgroup '%(new_wg)s'. \n"
-                "Some items failed, they had the id's: %(bad_ids)s"
-            ) % {
-                'new_wg': org_name,
-                'num_items': sucessfully_moved_items,
-                'bad_ids': ",".join(failed_items)
-            }
+                "{} items moved into the workgroup '{}'.\nSome items failed, they had the id's: {}".format(
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/forms/bulk_actions.py]: Revert plz.

**Comment by harrywhite4** [comment]
Currently waiting on #1574

**Comment by LegoStormtroopr** [comment]
Ready to roll.

**Pull request merged**